### PR TITLE
update(readme): fix invalid link

### LINKED ISSUE #20: [DOC] - invalid contact link
### What is incorrect in the documentation?

I may be contacted through my [Chat Server](http://chat.nhcarrigan.com) or via [email form](https://www.nhcarrigan.com/contact).
The email form doesn't work since the https://www.nhcarrigan.com/contact is removed.

### What should that text say?

I may be contacted through my [Chat Server](http://chat.nhcarrigan.com) or via [platforms](https://www.nhcarrigan.com/#contact).
It is basically the same but instead of /contact link to /#contact.

### Additional information
I edited via email form to platforms since there is no email form there. I had no idea whether to choose platforms or social media so I chose platforms.
_No response_

### PR DESCRIPTION
**Pull request by RawandDev**
# Pull Request

## Description:
Fix the invalid link from readme.
 

## Related Issue:

Closes #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,5 +29,5 @@ Looking for a repository and don't see it? I keep all of my old, deprecated proj
 <hr />
 <h2 align="center">Contact</h2>
 <p align="center">
-I may be contacted through my <a href="http://chat.nhcarrigan.com" target="_blank" rel="noopener noreferrer">Chat Server</a> or via <a href="https://www.nhcarrigan.com/contact" target="_blank" rel="noopener noreferrer">email form</a>.
+I may be contacted through my <a href="http://chat.nhcarrigan.com" target="_blank" rel="noopener noreferrer">Chat Server</a> or via <a href="https://www.nhcarrigan.com/#contact" target="_blank" rel="noopener noreferrer">other social platforms</a>.
```
Comment (nhcarrigan) [inline README.md:32]:
```suggestion
I may be contacted through my <a href="https://chat.nhcarrigan.com" target="_blank" rel="noopener noreferrer">Chat Server</a> or via <a href="https://www.nhcarrigan.com/#contact" target="_blank" rel="noopener noreferrer">other social platforms</a>.
```

This supports SSL now.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RawandDev) [reply README.md]: Done.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**